buoy.js

```diff
@@ -1,42 +1,66 @@
 /**
- * Buoy initializer.
+ * Buoy main "app" file.
+ *
+ * This file contains the Buoy front-end JavaScript "root" code; all
+ * other Buoy JavaScripts are dependent on this file, so this file
+ * must be present and loaded before any other dependent files.
+ *
+ * In this file, we create a global `BUOY` object using the (classic)
+ * {@link http://www.adequatelygood.com/JavaScript-Module-Pattern-In-Depth.html|JavaScript Module Pattern}
+ * and return a simple interface to interact with {@link http://BackboneJS.org|Backbone}
+ * library components (`Models`, `Views`, and `Collections`), alon with
```

Comment:
Yeah. `¯\_(ツ)_/¯` I'm gonna leave this here so that a future contributor who's less familiar with GitHub/git can see it, be annoyed by it, and try to fix it. Fixing typos inside of comments is a great "first patch" sort of exercise.